tools/frrcommon.sh.in

```diff
@@ -253,8 +253,8 @@ all_start() {
 all_stop() {
 	local pids reversed
 
-	daemon_list daemons disabled
-	[ "$1" = "--reallyall" ] && daemons="$daemons $disabled"
+	daemon_list daemons disableddaemons
```

Comment:
Overall, looks good to me, just for _consistency_ I suggest naming them the same as in: `daemon_list enabled_daemons disabled_daemons` (`frrinit.sh.in`).

Reply:
done